src/DefaultUser.php

```diff
@@ -47,18 +51,24 @@ public function getIdentity() : string
         return $this->identity;
     }
 
+    /**
+     * @psalm-return array<int|string, string>
+     */
     public function getRoles() : array
     {
         return $this->roles;
     }
 
+    /**
+     * @psalm-return array<string, mixed>
+     */
     public function getDetails() : array
     {
         return $this->details;
     }
 
     /**
-     * @param mixed $default Default value to return if no detail matching
+     * @param mixed|null $default Default value to return if no detail matching
```

Comment:
`mixed|null` is equivalent to `mixed` (`mixed` contains the `null` type)

Reply:
Fixed.